src/Illuminate/View/ClassLessComponent.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Illuminate\View;
+
+class ClassLessComponent extends Component
+{
+    /**
+     * The component view.
+     *
+     * @var string
+     */
+    protected $view;
+
+    /**
+     * The component data.
+     *
+     * @var array
+     */
+    protected $data = [];
+
+    /**
+     * Create a new class-less component instance.
+     *
+     * @param  array  $data
+     * @return void
+     */
+    public function __construct($data)
```

Comment:
I wanted to pass the view as a separate parameter but couldn't immediately figure out how. Will need to be cracked in the `componentString` method of the `ComponentTagCompiler` class.

Reply:
Solved this 👍